daemon-tests/tests/happy_path.rs

```diff
@@ -175,6 +175,37 @@ async fn taker_takes_order_and_maker_rejects() {
     wait_next_state!(received.id, maker, taker, CfdState::Rejected);
 }
 
+#[tokio::test]
+async fn another_offer_is_automatically_created_aftertaker_takes_order() {
+    let _guard = init_tracing();
+    let (mut maker, mut taker) = start_both().await;
+
+    // TODO: Why is this needed? For the cfd stream it is not needed
+    is_next_none(taker.order_feed()).await.unwrap();
+
+    maker.publish_order(dummy_new_order(Position::Short)).await;
```

Comment:
We might want to rename this to something like `set_offer_parameters`?

Reply:
sounds good, changed this and also changed `NewOrder` to `OfferParams`